tools/conf/ol3/ol-custom.json

```diff
@@ -61,7 +61,10 @@
       "externs/olx.js",
       "externs/proj4js.js",
       "externs/tilejson.js",
-      "externs/topojson.js"
+      "externs/topojson.js",
+	  "externs/bingmaps.js",

```

Comment:
I think these have been removed previously by design to minimize the size. Some reason to add these?